packages/jest/src/migrations/update-8-3-0/update-8-3-0.ts

```diff
@@ -15,18 +15,23 @@ function updateJestConfig(host: Tree) {
       ts.ScriptTarget.Latest
     );
     const changes: RemoveChange[] = [];
-
-    getSourceNodes(sourceFile).forEach(node => {
+    const sourceNodes = getSourceNodes(sourceFile);
+    sourceNodes.forEach((node, index) => {
       if (
         ts.isPropertyAssignment(node) &&
         ts.isIdentifier(node.name) &&
         node.name.text === 'collectCoverage'
       ) {
+        const expectedCommaNode = sourceNodes[index + 4];
```

Comment:
@rarmatei  I'm just curious, how is this getting the possible comma node? -> `sourceNodes[index+4]` What's the purpose of  `+4` ? thx in advance!

Reply:
thanks for the question, Martin! 

I basically noticed that in the type of file we were targeting for the migrations (jest.config.js), if it finds a node that matches `ts.isPropertyAssignment(node) && ts.isIdentifier(node.name)`, its follow-up comma node (if it exists) will be at position `+4` from the current node's index:

```js
module.exports = {
  //...
 clearMocks: true, // <-- if "clearMocks" has index 10 (for example), the comma will be at index 14
 collectCoverage: 'coverage'
};
```

**This is definitely not the most elegant solution** - what we really need to look at is fixing the *removal* case in `ast-utils.ts` (pls read my comment above for context: https://github.com/nrwl/nx/pull/2442#discussion_r375993993)

But it provided a quick fix to ensure these migrations will not break jest.config files that have the options we're looking to remove either at the end (not followed by a comma), or somewhere in the middle or at the top (followed by a comma)

But let me know if you noticed any bugs with this or if you have ideas for a better solution and I can have a look!! Thanks for all your help with `Nx` 💯

Reply:
> what we really need to look at is fixing the removal case in ast-utils.ts 

gotcha, I understood that, just wanted to double check if that `4` is for `true` :)
- awesome explanation/communication btw 👌

**Why I was asking:**
I'm writing schematic rule that removes some default settings from `@nrwl` generators, for our purposes as we migrate to `nx`. One of these removals is to remove `transform` field, as we are/and will use babel-jest(performance,ecosystem). 

So I wanted to implement similar "hack", but was not sure what's the best approach haha, as the property value is an `ObjectLiteralExpression` token - was wondering how to effectively get that length number. Any ideas ?

![image](https://user-images.githubusercontent.com/1223799/76325540-b57c2180-62e7-11ea-9bb8-ee06b410a267.png)

cheers!

Reply:
Ah, I see!

hmm, if you look in these migrations files, it [sends the **whole** text of the node to remove from the file](https://github.com/rarmatei/nx/blob/28438ff19fd3f5165b59779251f8660bed1432a5/packages/jest/src/migrations/update-8-3-0/update-8-3-0.ts#L34) (`getText / getFullText`)

I would have assumed that regardless of the property value, it always sends the whole text, so the length you mark for removal is always correct. Is that not the case for your object then?

Reply:
yeah I'm using that, just wanted to prevent the same issue with possible `comma` .
![image](https://user-images.githubusercontent.com/1223799/76337485-b61cb400-62f7-11ea-851c-0bbe5f4e6394.png)

hey no worries you've already helped me ! 

thanks a lot 🙌